bdsx/event_impl/blockevent.ts

```diff
@@ -75,18 +142,6 @@ function onBlockPlace(blockSource:BlockSource, block:Block, blockPos:BlockPos, f
 }
 const _onBlockPlace = procHacker.hooking("BlockSource::mayPlace", bool_t, null, BlockSource, Block, BlockPos, int32_t, Actor, bool_t)(onBlockPlace);
 
-export enum PistonAction {
```

Comment:
Please keep backward compatibility.
If you just delete them. It can break other plugins.